packages/workbox-core/_default.mjs

```diff
@@ -61,6 +62,8 @@ class WorkboxCore {
         `${padding}https://github.com/GoogleChrome/workbox/issues/new`
       );
       logger.groupEnd();
+
+      checkSWFileCacheHeaders();
```

Comment:
Small nit: could you wrap this in

```js
if (typeof checkSWFileCacheHeaders === 'function') {
  checkSWFileCacheHeaders();
}
```

just to double-check that we're not somehow in a scenario in which it's set to `null`? I can't really imagine why that would happen with our normal builds, since this is in a `NODE_ENV !== 'production'` block , but I wonder if some custom build scenarios might end up leading to a mismatch.